featurebyte/service/online_enable.py

```diff
@@ -235,18 +195,16 @@ async def update_feature(
                 data=FeatureServiceUpdate(online_enabled=online_enabled),
                 document=document,
                 return_document=True,
+                populate_remote_attributes=False,
             )
             assert isinstance(feature, FeatureModel)
             await self._update_feature_namespace(
                 feature_namespace_id=feature.feature_namespace_id,
                 feature=feature,
-                return_document=False,
             )
-            for feature_list_id in feature.feature_list_ids:
-                await self._update_feature_list(
-                    feature_list_id=feature_list_id,
-                    feature=feature,
-                    return_document=False,
-                )
-
-            return await self.feature_service.get_document(document_id=feature_id)
+            await self._update_feature_lists(
+                feature_list_ids=feature.feature_list_ids,
+                feature=feature,
+            )
+
+        return await self.feature_service.get_document(document_id=feature_id)
```

Comment:
This is the key update of this PR. Move the final `feature_service.get_document` out of the transaction context. Revise `_update_feature_namespace` and `update_feature_list` methods to avoid redundant query & deserialization.